Source/WTF/Scripts/GeneratePreferences.rb

```diff
@@ -96,6 +97,7 @@ def initialize(name, opts, frontend)
     @type = opts["type"]
     @refinedType = opts["refinedType"]
     @status = opts["status"]
+    @category = opts["category"]
```

Comment:
nit: I think I'd prefer defaulting to "none" here instead of in the getter methods.
```suggestion
    @category = opts["category"] || "none"
```

Reply:
Oh, that's a great idea -- I'll adopt that.